src/constants/colors/index.js

```diff
@@ -132,10 +151,16 @@ export const guideColors = {
   recommendedCleansersGuide: '#cad1b5',
 };
 
-export default Object.assign(
-  {},
-  brandColors,
-  colorAliases,
-  postcardColors,
-  guideColors
-);
+const colorsCompilation = {
+  ...brandColors,
```

Comment:
you can add `...legacyColors` for now so that we can still have purple80/60 exposed as our color constants

Reply:
ok fixed